### PR TITLE
Revert "Revert "Attempts to clarify enrollment mode, timeframe, cert result""

### PR DESCRIPTION
**Pull request by lamagnifica**
Reverts edx/edx-documentation#820
Should resurrect PR 814

### CODE REVIEW

**Comment by lamagnifica** [comment]
@peter-fogg and @marcotuts  and @stroilova please look at this one -- it resurrected PR #814

**Comment by peter-fogg** [comment]
https://www.youtube.com/watch?v=wZv62ShoStY#t=3m09s

Looks good to me :+1:

File: en_us/data/source/internal_data_formats/sql_schema.rst [hunk 1]
```diff
@@ -811,12 +811,18 @@ mode
 
   **History**:
 
-  * All enrollments prior to 20 Aug 2013 are "honor", when the "audit" and
-    "verified" values were added.
+  * On 1 Dec 2015, the "audit" value was reintroduced. This value now
```
Comment (stroilova) [inline en_us/data/source/internal_data_formats/sql_schema.rst:814]: @peter-fogg  @marcotuts  I recall the HC/FA audit mode was introduced "in Dec 2015".

Could you please confirm the date and details: are new audit enrollments in all courses created after e.g. Dec 1, 2015 not eligible for certs?

**Comment by stroilova** [comment]
Thanks @lamagnifica ! This looks good modulo checking the HC/FA details & date with Miki or Marco.

File: en_us/data/source/internal_data_formats/sql_schema.rst (hunk 1 above)
Comment (peter-fogg) [inline en_us/data/source/internal_data_formats/sql_schema.rst:814]: @stroilova Looking back at the Git/release history, 12/1/2015 is correct. All audit enrollments after that date are ineligible for certficates.

**Comment by lamagnifica** [comment]
@stroilova please spell out what you mean by HC/FA.
@mikigoyal or @marcotuts please provide details and dates per Olga's request.

**Comment by marcotuts** [comment]
@peter-fogg saves the day with date details described directly in the document. how delightful.

**Comment by lamagnifica** [comment]
@stroilova based on Marco's comment above and Mik's comment at the end of this wiki page, is this good to go now? https://openedx.atlassian.net/wiki/display/ENG/2016-02-04

**Comment by stroilova** [comment]
Great! Thanks for tracking that down -- let's get this in. :+1:

**Pull request merged**